modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-production-deployment.adoc

```diff
@@ -328,25 +332,70 @@ Install the Redpanda Helm chart in a different namespace to your existing Redpan
 
 NOTE: Make sure to use unique node ports for the listeners in your Redpanda resource so that they don't conflict with any existing node ports in your other Redpanda clusters. See <<External access>>.
 
-```bash
+[source,bash,subs="attributes+"]
+----
 helm repo add redpanda https://charts.redpanda.com
 helm install redpanda-two redpanda/redpanda \
+  --version {latest-redpanda-helm-chart-version} \
   --namespace <anothernamespace> \
   --set listeners.kafka.external.default.advertisedPorts[0]=31093 \
   --set listeners.admin.external.default.advertisedPorts[0]=31645 \
   --set listeners.http.external.default.advertisedPorts[0]=30083 \
   --set listeners.rpc.port=33146 \
   --set listeners.schemaRegistry.external.default.advertisedPorts[0]=30084
   --create-namespace
-```
+----
 --
 
 ====
 
+[[config]]
 == Production considerations
 
 This section provides advice for configuring the Redpanda Helm chart for production. For all available settings, see xref:reference:k-redpanda-helm-spec.adoc[]. To learn how to customize the Redpanda Helm chart, see xref:manage:kubernetes/k-configure-helm-chart.adoc[Customize the Helm Chart].
 
+[[version-pinning]]
+=== Version pinning
```

Comment:
Can this paragraph/section include a reference to release notes page modules/reference/pages/releases/index.adoc?

It could say that any major changes that requires reconfiguration or config adjustment could be found in e.g. https://deploy-preview-477--redpanda-docs-preview.netlify.app/current/reference/releases/